### ISSUE TITLE
libxt: fatal error: glib.h: No such file or directory

### ISSUE THREAD

**Issue by thackl** [maxim-belkin/homebrew-xorg#460]
**Please note we will close your issue without comment if you delete, do not read or do not fill out the issue checklist below and provide ALL the requested information. If you repeatedly fail to use the issue template, we will block you from ever submitting issues to Homebrew again.**

- [x] have a problem with `brew install` (or `upgrade`, `reinstall`) a single formula? If it's a general `brew` problem please file this issue at Linuxbrew/brew: https://github.com/Linuxbrew/brew/issues/new/choose. If it's a tap (e.g. Brewsci/homebrew-bio) problem please file this issue at the tap.
- [x] ran `brew update` and can still reproduce the problem?
- [x] ran `brew doctor`, fixed all issues and can still reproduce the problem?
- [ ] ran `brew gist-logs <formula>` (where `<formula>` is the name of the formula that failed) and included the output link?
- [x] if `brew gist-logs` didn't work: ran `brew config` and `brew doctor` and included their output with your issue?

I ran `brew install r`and it failed during compilation with "fatal error: glib.h: No such file or directory". I checked and glib is installed and linked properly 
```
brew link glib
Warning: Already linked: /home/thackl/.linuxbrew/Cellar/glib/2.58.1"
```
Also in the command that failed, `-I/path/to/glib/include/` is set correctly and points to `glib.h`.
[brew install log and brew config](https://gist.github.com/thackl/15b1d65065d99bce703b107d7f4913d1)

Any idea why this doesn't work? FYI, I just started using Linuxbrew, good chance the problem is on my side. I use linuxbrew as non-root.

**Comment by maxim-belkin** [comment]
@Martingales Could you please try 

1. Updating `Linuxbrew` to the latest version and building `libxt` again?
2. If (1) does not work, try building `libxt` without compile-time tests like so

```
brew install --without-test libxt
``` 

I was not able to reproduce the bug in a Linuxbrew container based on Ubuntu system (see the link below). I'll try to see if it is reproducible in a different container.
https://gist.github.com/59575ae114d7791dc6878099ba5b4920

**Comment by maxim-belkin** [comment]
> 40-core 64-bit haswell

woo-hoo!

**Comment by maxim-belkin** [comment]
@iMichka, at least in CentOS 7, `r` requires `perl` at build time (because there is no system perl)

**Comment by maxim-belkin** [comment]
And `pcre2` is an undeclared dependency with linkage

**Comment by sjackman** [comment]
@jaredsampson Try running… `xargs -0 file --no-dereference --print0 </dev/null`
If that fails, that's the source of trouble.

See https://github.com/Homebrew/brew/blob/2ff07a60c8fe5c95e1648b368f65e5dccf3b6d46/Library/Homebrew/keg_relocate.rb#L140-L141

**Comment by sjackman** [comment]
I've opened PR https://github.com/Linuxbrew/homebrew-xorg/pull/461 to hopefully address this issue.

**Comment by maxim-belkin** [comment]
@thackl and @jaredsampson, could you please check if this issue has been resolved by #461 for you?

**Comment by jaredsampson** [comment]
@maxim-belkin @sjackman After the homebrew 2.0 migration, and I suspect due to the fix in #461, yes, I have been able to install `libxt` successfully with a simple `brew install`.  Thanks again!

**Comment by sjackman** [comment]
Thanks for following up, Jared!

Apropos of nothing, a colleague of mine is named Jared Simpson https://github.com/jts
